### ISSUE TITLE
Unable to handle JsonNode

### ISSUE THREAD

**Issue by alfisti166** [organics2016/pojo2json#24]
Plugin seems not to handle classes that includes JsonNode field. Stack overflow occurs.

```java
public class Asd {
    private JsonNode asd;
}
```
Is it possible just to generate empty brackets?

**Comment by organics2016** [comment]
This is a bug. fix to next release

**Comment by organics2016** [comment]
![idea64_iWBNwhsREa](https://user-images.githubusercontent.com/16286533/226114948-081465cf-9fed-47f3-9be5-b6775d2c2e88.png)

This is my test case. 

The result shows that although the value is not as expected, but it does not cause a stack overflow.
Did you stuff something weird in your Class?
eg:
```
public class Asd extends JsonNode {
    private JsonNode asd;
}
```
or
```
public class JsonNode {
    private Asd asd;
}

public class Asd {
    private JsonNode asd;
}
```

**Comment by alfisti166** [comment]
Not really. Let me provide more details.
BTW. ObjectNode doesn't cause stack overflow, but generate pretty useless fields.


![Screenshot from 2023-03-18 16-45-03](https://user-images.githubusercontent.com/18468217/226116115-7ccb272b-8298-4056-a97f-f024aad3e372.png)
![Screenshot from 2023-03-18 16-37-38](https://user-images.githubusercontent.com/18468217/226116117-5efa5c3e-0910-48e2-976e-f772ba3ce3ed.png)
![Screenshot from 2023-03-18 16-37-04](https://user-images.githubusercontent.com/18468217/226116118-fbf5f48a-add0-4e00-aba8-dc28d08ffbbf.png)

**Comment by organics2016** [comment]
Moderation may take up to two business days. 
Fix version 1.2.6
